js/plates-model/field.js

```diff
@@ -251,6 +259,10 @@ export default class Field extends FieldBase {
         this.volcanicAct = new VolcanicActivity(this)
       }
       this.volcanicAct.setCollision(field)
+      if (this.isOcean && field.isContinentBuffer) {
```

Comment:
Is this check necessary in addition to the one above?

Reply:
Which one above? The similar one in the first `if` statement? 
If so, then yes, it's necessary as one would apply force to the top plate, and the second one to the bottom plate.

Reply:
OK I see, they both drag each other. Is this preferable to adding `field.draggingPlate = this.plate` in the first `if` statement?

Reply:
It's a bit more complicated. Note that collideWith handles only side of collision. As currently field A can collide with field B, but B might not collide with field A (due to optimized collision detection). So, yes, we could do what you suggested, but it'll be just a bit different behaviour. Not better, not worse.
 
But it's very reasonable question and it made me start working on refactoring of this part. I got to some way cleaner implementation where collision is handled for the pair (not single field like now). But it works a bit different and I'm currently testing if it's a problem or not. I'll update this PR when it's ready.